docs/_includes/Configuring-Alluxio-with-GCS/gcs-file.md

```diff
@@ -1 +1,3 @@
-    GCS_BUCKET/GCS_DIRECTORY/alluxio/data/default_tests_files/Basic_CACHE_THROUGH
+```
```

Comment:
I treated these as non-property code blocks. Are these types of references in a property or config file?